app/services/witter.js

```diff
@@ -0,0 +1,7 @@
+const axios = require('axios');
+
+const GEEK_JOKES_URL = 'https://geek-jokes.sameerkumar.website/api?format=json';
```

Comment:
Use env vars to set external services endpoints. In `app/index` you will find the config.

Reply:
Solved